testing/scenario_app/run_android_tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
```

Comment:
I'd prefer that we put all the test harnesses into `run_tests.py` instead of having extra bash scripts lying around. Is there a reason to keep this separate, or is it clear to move there?

Reply:
Is this the one question that you mentioned in your LGTM?

Reply:
Yeah.

Reply:
Ok. I actually don't know why this is here, or what it's accomplishing. If you can guide me to moving some/all of this to another place, I can do that.

Reply:
It looks like it builds libapp.so with `compile_android_aot.sh`, and then runs `./gradlew assembleAndroidTest && ./gradlew connectedAndroidTest`.

I think you could just basically convert the bash commands into `subprocess.check_call` ones in [run_tests.py](https://github.com/flutter/engine/blob/master/testing/run_tests.py), with a new `AndroidIntegration` test type to run these under.

The reason I'm suggesting putting it together is just so that all the tests are in one place, but maybe there's some semantic reason to keep the android e2e tests separate? /cc @dnfield and @xster.

Reply:
Ideally more of this would just be in the build rules, but I haven't had time to get to that. Moving it into run_tests.py is probably fine but it would be best if the build just produced the relevant artifacts.

Reply:
(I am, of course, fine with any improvements made to this)

Reply:
Let's assume I'm an idiot when it comes to bash scripts and interfacing with Python (not really an assumption). @mklim can you outline the approach? There are a lot of Python methods in run_tests.py that my IDE is saying have no usages. Is there some kind of convention name matching going on?

Reply:
BTW, would there be any input configurations? For example, java tests let you choose the test, but does it make sense to have any control over which tests runs in the integration case?

Reply:
Hmm. I've never tried to edit python in a full IDE before, not sure how it would tell what isn't used. I'm not exactly a python expert either.

I think if the ideal is that we want to delete this eventually, we should just leave it here for now instead of moving it. There's not really a lot of point of refactoring it into python if we don't want it around for long. I can sync up with you on how to translate this if you'd like Matt, but with Dan's input I think it's better to just leave it alone after all.

Reply:
Ok. I'll leave it as-is. Any other comments on the PR at this point?

Reply:
No, LGTM. :)